### PR TITLE
adding site to analytics manager and adding context to ga onStartup.

### PR DESCRIPTION
**Pull request by cchatham**

### CODE REVIEW

File: include/system/analytics_manager.js [hunk 1]
```diff
@@ -130,44 +140,54 @@ module.exports = function AnalyticsManagerModule(pb) {
...
      * @return {Boolean} TRUE if was unregistered, FALSE if not found
      */
-    AnalyticsManager.unregisterProvider = function(name) {
-        if (!AnalyticsManager.isRegistered(name)) {
+    AnalyticsManager.unregisterProvider = function(name, site) {
+        if (!AnalyticsManager.isRegistered(name, site)) {
             return false;
         }
-        delete PROVIDER_HOOKS[name];
+        delete PROVIDER_HOOKS[site][name];
```
Comment (brianhyder) [inline include/system/analytics_manager.js]: In order to remain backward compatible we need to default site to global here if it is not provided.

**Comment by brianhyder** [comment]
Good job on this @cchatham!   One small change and we're ready to merge.

**Comment by brianhyder** [comment]
Fixes #945

File: include/system/analytics_manager.js (hunk 1 above)
Comment (cchatham) [inline include/system/analytics_manager.js]: @brianhyder Sorry! missed one! good catch.

**Comment by cchatham** [comment]
@brianhyder fixed I think.

**Pull request merged**